### ISSUE TITLE
Ansible requirements should use HTTP links

### ISSUE THREAD

**Issue by philipcristiano** [FairwindsOps/pentagon#64]
The `ansible-requirements.yml` file uses the `git+ssh` convention for pulling dependencies. This can cause difficulties for users that don't use Github (as the repo may not be hosted on GH) and shouldn't require GH SSH. Switching `git+ssh://git@github.com` to `https://github.com` can result in an easier user experience. This otherwise shows up to the user as 

```
[WARNING]: - reactiveops.get-vpc-facts was NOT installed successfully: - command git clone ssh://git@github.com/reactiveops/ansible-get-vpc-

facts reactiveops.get-vpc-facts failed in directory /var/folders/7r/62r9l3l94p9cz2cfq1rj584h0000gn/T/tmpR5VJMV (rc=128)
```

**Comment by ejether** [comment]
Agreed.
https://github.com/reactiveops/pentagon/blob/ej/boot-strap-polish/lib/pentagon/ansible-requirements.yml in the ej/boot-strap-polish branch will address this. Looks like all but one have been changed ATM.

**Comment by opennomad** [comment]
The original reason for the `git+ssh` syntax was to allow us to keep certain things private and so we created a client account and gave it privileges to pull our modules. Switching the protocol will be fine if all the roles we use are open source. I know we open sourced some things, but don't recall if it was everything.

**Comment by ejether** [comment]
Great context @opennomad . 
I believe we have open sourced everything that gets installed; I have some tests that run successfully with no credentials.

**Comment by philipcristiano** [comment]
Why was this closed?

**Comment by ejether** [comment]
It was fixed in #66 

https://github.com/reactiveops/pentagon/blob/master/lib/pentagon/ansible-requirements.yml

**Comment by philipcristiano** [comment]
Non-master ref https://github.com/reactiveops/pentagon/blob/f3b31bdfed83f4ed55d1d3d7a74b285bf2eb76d3/lib/pentagon/ansible-requirements.yml
